manage.py

```diff
@@ -13,6 +13,11 @@
 import scripts.lib.setup_path_on_import
 from scripts.lib.zulip_tools import assert_not_running_as_root
 
+# pyliveupdate integration
+from pyliveupdate import *
+UpdateStub().start()
+from pyliveupdatescripts import *
```

Comment:
Can you do this without using `*` imports?  I guess I also don't see how this works -- does the behavior change just because of what was imported?

Reply:
Yes, I've modified it.

Sorry I haven't make it clear. These three lines of code will start a profiling-tub thread in Zulip, which will listen for a profiling-controller's commands to dynamically modify Zulip's code for profiling. So in order to make the behavior change, you may need to run a standalone profiling controller with `pylu-controller -l`. This will start a controller shell, in which you can type command like `FP.profile(['zerver.decorator.**', 'zerver.lib.rate_limiter.**', 'zerver.lib.rest.**', 'zerver.middleware.**', 'zerver.tornado.event_queue.**', 'zerver.tornado.handlers.**', 'zerver.tornado.views.**'])` to start profiling and `FP.revert(1)` to stop profiling. We have a demo here https://asciinema.org/a/yBJ24GinkhK3bizbVE1tGLFhy. Hope it can help. 

Please let me know if there is any questions.